examples/sampling/graphbolt/quickstart/node_classification.py

```diff
@@ -2,6 +2,7 @@
 This example shows how to create a GraphBolt dataloader to sample and train a
 node classification model with the Cora dataset.
 """
+
```

Comment:
ditto.

Reply:
i didn't modify this. I guess it is because i ran lintrunner, which changed the format of other code snippets.

Reply:
Can you just revert this?